docker-compose/docker-compose.test.yml

```diff
@@ -26,16 +27,47 @@ services:
     ports:
       - "5000:80"
     depends_on:
-      - ms-sql-server
-      - rabbitmq
-    volumes:
-      - ../:/app
+      ms-sql-server:
+        condition: service_started
+      rabbitmq:
+        condition: service_started
+      seed-mssql:
+        condition: service_completed_successfully
+      azure-storage-emulator:
+        condition: service_started
+    configs:
+      - source: ConsumerApiConfig
+        target: app/appsettings.json
+      - source: ConsumerApiOverrideConfig
+        target: app/appsettings.override.json
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost/health"]
+      test: ["CMD", "wget", "http://localhost/health"]
```

Comment:
Why did you remove the `start_period` from this healthcheck?

Reply:
`start_period` is the time Docker will wait before starting the healthchecks.
It came to my attention that in some scenarios (when testing locally and the consumer api is already built) the _booting up_ of the consumer api is rather quick. Moreover, the value we had before was more random than anything.

So, by removing it, we're performing healthchecks from the moment the container starts running. Considering it doesn't harm the loading process, and it's not a heavy operation, I figured it best to leave this way.

Reply:
I probably would have configured a 5-10s waiting time. But you're right, it probably doesn't harm.